modules/configuring-pods-static-ip.adoc

```diff
@@ -0,0 +1,44 @@
+// Module included in the following assemblies:
+//
+// * networking/ovn_kubernetes_network_provider/configuring-secondary-networks.adoc
+
+:_content-type: CONCEPT
+[id="configuring-pods-static-ip_{context}"]
+= Configuring pods with a static IP address
+
+The following example provisions a pod with a static IP address.
+
+[NOTE]
+====
+* You can only specify the IP address for a pod's secondary network attachment for a layer 2 or localnet attachment.
+* Specifying a static IP address for the pod is only possible when the attachment configuration does not feature subnets.
+====
+
+[source,yaml]
+----
+apiVersion: v1
+kind: Pod
+metadata:
+  annotations:
+    k8s.v1.cni.cncf.io/networks: '[
+      {
+        "name": "l2-network", <1>
+        "mac": "02:03:04:05:06:07", <2>
+        "interface": "myiface1",
+        "ips": [
+          "192.0.2.20/24"] <3>
+      }
+    ]'
+  name: tinypod
+  namespace: ns1
+spec:
+  containers:
+  - args:
+    - pause
+    image: k8s.gcr.io/e2e-test-images/agnhost:2.36
+    imagePullPolicy: IfNotPresent
+    name: agnhost-container
+----
+<1> The name of the network. This attribute is not namespaced.
+<2> The MAC address for the SR-IOV device that is allocated from the resource type defined in the SR-IOV network attachment definition CR.
+<3> Specify addresses for the SR-IOV device which is allocated from the resource type defined in the SR-IOV network attachment definition CR. Both IPv4 and IPv6 addresses are supported.
```

Comment:
Why are we mentioning SR-IOV devices ?

Reply:
@maiqueb Apologies, I copied this over. There isn't a definition/explanation that I could find for either `interface` or `ips` in the current docs. Would you provide one, **or** check these? 

Mac: <2> The MAC address to be assigned for the interface.

Interface: <3> The name of the network interface to be created for the pod.

IPS: <4> The IP addresses to be assigned to the network interface

Reply:
I think this description is accurate. 

Thank you.